browser-test/src/support/admin_programs.ts

```diff
@@ -381,8 +393,18 @@ export class AdminPrograms {
     return this.page.locator(`label:has-text("${Eligibility.IS_NOT_GATING}")`)
   }
 
-  async gotoEditDraftProgramPage(programName: string) {
+  async gotoEditDisabledDraftProgramPage(programName: string) {
+    await this.gotoEditDraftProgramPage(programName, true)
+  }
+
+  async gotoEditDraftProgramPage(
+    programName: string,
+    isDisabled: boolean = false,
```

Comment:
**suggestion, non-blocking, optional**: it took me a second to figure out what `isDisabled` means here. maybe naming it `programIsDisabled` or whatever is consistent with the file?

Reply:
yup made the change to `isProgramDisabled` to stay consistent with the file